src/network/room.cpp

```diff
@@ -48,6 +48,15 @@ class Room::RoomImpl {
     IPBanList ip_ban_list;             ///< List of banned IP addresses
     mutable std::mutex ban_list_mutex; ///< Mutex for the ban lists
 
+    struct PostOffice {
+        const u8 WaitPacket = 10; ///< Wait post office's reply max times.
```

Comment:
If this is always const, it can be turned into a static constexpr member, making the struct a little smaller.

```suggestion
        static constexpr u8 WaitPacket = 10; ///< Wait post office's reply max times.
```